runtime/brooklyn/src/ethereum.rs

```diff
@@ -29,122 +22,13 @@ pub const GAS_PER_SECOND: u64 = 40_000_000;
-			&account_id,
-			fee.unique_saturated_into(),
-			WithdrawReasons::FEE,
-			ExistenceRequirement::AllowDeath,
-		)
-		.map_err(|_| Error::<T>::BalanceLow)?;
-		Ok(Some(imbalance))
-	}
-
-	fn correct_and_deposit_fee(
-		who: &H160,
-		corrected_fee: U256,
-		base_fee: U256,
-		already_withdrawn: Self::LiquidityInfo,
-	) -> Self::LiquidityInfo {
-		if let Some(paid) = already_withdrawn {
-			let account_id = T::AddressMapping::into_account_id(*who);
-
-			// Calculate how much refund we should return
-			let refund_amount = paid
-				.peek()
-				.saturating_sub(corrected_fee.unique_saturated_into());
-			// refund to the account that paid the fees. If this fails, the
-			// account might have dropped below the existential balance. In
-			// that case we don't refund anything.
-			let refund_imbalance = C::deposit_into_existing(&account_id, refund_amount)
-				.unwrap_or_else(|_| C::PositiveImbalance::zero());
-
-			// Make sure this works with 0 ExistentialDeposit
-			// https://github.com/paritytech/substrate/issues/10117
-			// If we tried to refund something, the account still empty and the ED is set to 0,
-			// we call `make_free_balance_be` with the refunded amount.
-			let refund_imbalance = if C::minimum_balance().is_zero()
-				&& refund_amount > C::Balance::zero()
-				&& C::total_balance(&account_id).is_zero()
-			{
-				// Known bug: Substrate tried to refund to a zeroed AccountData, but
-				// interpreted the account to not exist.
-				match C::make_free_balance_be(&account_id, refund_amount) {
-					SignedImbalance::Positive(p) => p,
-					_ => C::PositiveImbalance::zero(),
-				}
-			} else {
-				refund_imbalance
-			};
-
-			// merge the imbalance caused by paying the fees and refunding parts of it again.
-			let adjusted_paid = paid
-				.offset(refund_imbalance)
-				.same()
-				.unwrap_or_else(|_| C::NegativeImbalance::zero());
-
-			let (base_fee, tip) = adjusted_paid.split(base_fee.unique_saturated_into());
-			// Handle base fee. Can be either burned, rationed, etc ...
-			OU::on_unbalanced(base_fee);
-			return Some(tip);
-		}
-		None
-	}
-
-	fn pay_priority_fee(tip: Self::LiquidityInfo) {
-		// Default Ethereum behaviour: issue the tip to the block author.
-		if let Some(tip) = tip {
-			let account_id = T::AddressMapping::into_account_id(<Pallet<T>>::find_author());
-
-			let tip_commission = TREASURY_COMMISSION_FROM_TIP * tip.peek();
-			let (tip_commission, tip_after_commission) = tip.split(tip_commission);
-			OU::on_unbalanced(tip_commission);
-
-			let _ = C::deposit_into_existing(&account_id, tip_after_commission.peek());
-		}
-	}
-}
-
 parameter_types! {
 	pub BlockGasLimit: U256 = U256::from(
 		NORMAL_DISPATCH_RATIO * BlockWeights::get().max_block.ref_time() / WEIGHT_PER_GAS
 	);
 	pub PrecompilesValue: GoldenGatePrecompiles<Runtime> = GoldenGatePrecompiles::<_>::new();
 	pub WeightPerGas: Weight = Weight::from_parts(WEIGHT_PER_GAS, 0);
-	pub ChainId: u64 = 888866;
+	pub ChainId: u64 = 8886;
```

Comment:
chain id for brooklyn is not decided, but 8886 is for sydney

Reply:
good catch.
fixed.